R/ir_analysis_query_builder.r

```diff
@@ -930,6 +961,9 @@ buildAnalysisQuery <- function(analysisExpression, analysisId, dbms, cdmSchema,
   for(i in seq_along(analysisExpression$strata)){
     strata <- analysisExpression$strata[[i]]
     cg <- strata[[1]]$expression
+    if (is.null(cg)){
```

Comment:
Could you pls double check why `strata[[1]]` is applied only?

Reply:
the strata is just an object here, I know that any value in R is a vector of one value, but for some reason  strata[[1]] returns nothing here.